README.md

```diff
@@ -52,29 +56,85 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 ![CoatRack architecture overview](./spring-boot/admin/src/main/resources/static/images/coatrack-architecture-overview.png)
 
-## Install
 
-Scripts for building and running CoatRack are available in the root folder of the code base. To build CoatRack, use:
 
-```console
-./build.sh
+## Installation using production-ready Docker Images
+
+ The following prerequisites are required:
+
+*  a Linux shell - Windows users can use WSL of the Git Bash shell instead
+*  docker
+*  docker-compose
+
+
+
+### Deployment For Newcomers
+
+This approach is suggested if you are completely new to CoatRack and want to just want to experience what it actually does. Follow these instructions:
+
+1. Change the `INSERT_SAMPLE_DATA_ON_STARTUP` parameter in `.env` to `true` to initialize the example Gateway in the database.
+2. Go to the directory `docker/docker-compose-deployment` and execute:
+
+```sh
+bash initialize-databases-if-necessary.sh
 ```
-To run the CoatRack web application, use:
 
-```console
-./run-admin-application.sh
+* The previous step is required to create the databases within PostgreSQL. Now you can set up CoatRack via:
+
+```sh
+docker-compose --profile example-gateway up -d
 ```
-After starting up, the CoatRack web application will be accessible at `http://localhost:8080` 
 
-*Optional:* If you are a CoatRack developer and want to run the application using a custom Spring profile, you can pass the profile name to the run script as follows:
+* The last step is to check out the service provided by the example Gateway. Search for this URL in your browser:
 
-```console
-./run-admin-application.sh [optional-spring-profile-name]
+```http
+http://localhost:8088/humidity-by-location?api-key=ee11ee22-ee33-ee44-ee55-ee66ee77ee88
 ```
-To create your personal custom Spring profile for CoatRack, please follow the instructions in the following template file:
+
+* The Example-Gateway should accept the provided API key and redirect you to the website of the `humidity-by-location` service.
+* CoatRack can be shutdown by just executing:
+
+```sh
+docker-compose --profile example-gateway down
 ```
-spring-boot/admin/src/main/resources/application-private-TEMPLATE.yml
+
+* If you want to remove CoatRack and all associated traces, execute:
+
+```sh
+bash stop-containers-and-clean-up-traces-if-existent.sh
+```
+
+
+
+### Deployment for Production
+
+This approach is meant for developers and people who would like to deploy a production-ready CoatRack instance with a clean, empty database and without an Example-Gateway. Just go to the directory `docker-compose-deployment` and execute:
+
+```sh
+bash initialize-databases-if-necessary.sh
+docker-compose up -d
```

Comment:
This approach pull the latest images build on master and not this branch. The instruction should work, when we merge this branch and push the images to Dockerhub.

Reply:
Should we then maybe move these instructions to a follow-up issue, so that can first merge this PR (without these instructions), then we can test these instructions properly before merging? This way we prevent that the master README contains instructions that have not yet been fully tested.

Reply:
Done as suggested. See comments below.